jwst/assign_wcs/tests/test_nirspec.py

```diff
@@ -57,7 +57,7 @@ def create_hdul():
     phdu.header['instrume'] = 'NIRSPEC'
     phdu.header['detector'] = 'NRS1'
     phdu.header['time-obs'] = '8:59:37'
-    phdu.header['date-obs'] = '2014-09-05'
+    phdu.header['date-obs'] = '2016-09-05'
```

Comment:
Actually the test `correct_tilt` has a dict as a `disperser` so I take back my comments above. Apparently CV3 ref files are used. 
@jdavies-st Could you disable now the nirspec tests and merge this PR. Thanks for the work!